test_integration/helpers/HelperInterface.ts

```diff
@@ -0,0 +1,20 @@
+import BigNumber from 'bignumber.js';
+import { TransactionReceipt } from 'web3-core';
+import { OSTPrime } from '@openst/mosaic-contracts/dist/interacts/OSTPrime';
+import { UtilityToken } from '@openst/mosaic-contracts/dist/interacts/UtilityToken';
+
+export interface HelperInterface {
```

Comment:
Same comments as other interface.
Documentation and license. Also new line in the end.
We should explain why this interface is needed.

Reply:
done